### ISSUE TITLE
Wrong conditional in createSourcemap variable?

### ISSUE THREAD

**Issue by marcofugaro** [jakemmarsh/angularjs-gulp-browserify-boilerplate#123]
Little issue I noticed while I was reading your code, really minor so feel free to close this issue if you want.

On the line https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/blob/master/gulp/tasks/styles.js#L14

`const createSourcemap = !global.isProd || config.styles.prodSourcemap;`

shouldn't you create the sourcemap only if it's not production AND I set true the config variable?

so sould the condition be:

`const createSourcemap = !global.isProd && config.styles.prodSourcemap;`

correct me if I'm wrong, thanks.

**Comment by DrewML** [comment]
For reference, the [Browserify task](https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate/blob/master/gulp/tasks/browserify.js#L54) uses the logic you recommend above:

```javascript
const createSourcemap = global.isProd && config.browserify.prodSourcemap;
```

However, I actually think the logic in the `Styles` task is "more correct".

**Styles Task**:
- If *not* building for production, use sourcemaps
- (If building for production) If prodSourcemap enabled, use sourcemaps

**Browserify Task**
- If *not* building for production and production sourcemaps enabled, use sourcemaps

I can't think of a scenario where you wouldn't want sourcemaps when working locally.

**Comment by marcofugaro** [comment]
Yeah thanks, my bad I didn't get that `prodSourcemap` actually stands for "production Sourcemap", I thought you were supposed to switch off the sourcemap in development to earn those 500ms over browserSync, but I realize now that makes no sense :)

Yes I think you should enable dev sourcemaps in the browserify task as well if you use some js preprocessor. You guys decide, feel free to close this issue after.

Thanks for explaining!

**Comment by DrewML** [comment]
No problem, and thanks for the heads-up @marcofugaro. It turns out, your observation helped me spot a bug with how we're currently handling sourcemaps (they're broken in development on first build).

**Comment by DrewML** [comment]
Fix for `Browserify` task included in PR #124.

**Comment by jakemmarsh** [comment]
Merged #124
